2019-07-13-StreamTest/src/CodeTest.java

```diff
@@ -0,0 +1,64 @@
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+
+public class CodeTest {
+
+	@Test
+	public void getAddress() { // #1
+		Person p1 = new Person("John", 37, new Address("Banana str.", 1));
+		Person p2 = new Person("Sarah", 14, new Address("Zwibeln str.", 4));
+		Person p3 = new Person("Bob", 25, new Address("Apfel str.", 7));
+		List<Person> persons = Arrays.asList(p1, p2, p3);
+		List<Address> actual = Code.getAddress(persons);
+		List<String> expect = Arrays.asList("Banana str.1, Apfel str.7");
```

Comment:
Here you create a list with one element, which is "Banana str.1, Apfel str.7", but from the junit console you may see that something different is expected, something like "["Banana str.1", "Apfel str.7"]" and this should be expected value.